mstm/__init__.py

```diff
@@ -40,9 +40,11 @@
 import os
 import shutil
 import numpy as np
+from scipy import interpolate
+from scipy import integrate
 
 # change to match the filename of your executable
-MSTM_EXE = 'mstm.exe'
+MSTM_EXE = 'mstm_ubuntu.exe'
```

Comment:
no - it makes sense to have the temp directory for these files, since they are truly temporary.  I just don't think that it makes sense to copy the executable to the temp directory.